.github/workflows/test.yml

```
@@ -0,0 +1,18 @@
+name: Backed Deployment
```

The typo here could probably be fixed.

---

If you can rename to a name like:
```suggestion
name: Multi threading Load Testing
```
A comment along side on what exactly this workflow do might help the developers who work on #18